### DISCUSSION TITLE
Hitsound sampleset and volume gets reset for every new hitobject

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#18725] [Q&A] [answered]
I don't know if this is intentional or a bug, but I find this very irritating. I would have expected it to work in a similar way to how slider velocity works. e.g. the next slider you place inherits the velocity of the one before it.

https://user-images.githubusercontent.com/22381687/173999829-31ee8e4c-b928-4f54-82af-47b2c6151428.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/18742
